code/modules/mob/living/basic/farm_animals/bee/_bee.dm

```diff
@@ -0,0 +1,338 @@
+#define BEE_TRAY_RECENT_VISIT 200 //How long in deciseconds until a tray can be visited by a bee again
```

Comment:
```suggestion
#define BEE_TRAY_RECENT_VISIT 20 SECONDS //! How long until a tray can be visited by a bee again
```

no need for that un-necessary portion of the comment, we can nest defines like this. the `//!` is so it can be dmdocced, but would be nice if you could do a more traditional format for all of these.